accounts/models.py

```diff
@@ -26,6 +36,14 @@ class CustomUser(AbstractUser):
         choices=LMS_MODULES_CHOICES
     )
 
+    organisation = models.ForeignKey(
+        Organisation,
+        on_delete=models.CASCADE,
+        related_name='user_organisation',
+        default='',
```

Comment:
That's only needed for Char or TextField and also you have `null=True` set as well.

Reply:
Removed `null=True` and refactored `default=''` to take in default ID of 1 based on Organisation Object display name. Seeding the data from the organisation.json file first to ensure this property works as expected.

https://stackoverflow.com/questions/9311996/setting-default-value-for-foreign-key-attribute/40813536#40813536 Resolution drafted from here with your help Stefan! Thank you for that.